content/blog/2022/2022-08-25-schedule-cron-job-in-node.md

```diff
@@ -0,0 +1,277 @@
+---
+title: "Schedule Job On Node.js"
+categories: ["Node"]
+date: 2022-08-25 00:00:00 +1100
+authors: ["ajibade"]
+description: "Quick guide to scheduling cron jobs in Node.js applications"
+image: images/stock/0031-matrix-1200x628-branded.jpg
+url: schedule-cron-job-in-node
+---
+
+Have you ever wanted to perform a specific task on your application server at specific times without physically running them yourself? Or you'd rather spend your time on more important tasks than remembering to periodically clear or move data from one part of the server to another. Cron Job schedulers can be used to achieve this easily.
+
+Cron job scheduling is a common practice in applications; it provides a level of autonomy in our applications.
+
+## Using node cron In node.Js
+In this article, we'll use the [node-cron](https://www.npmjs.com/package/node-cron) library to schedule cron jobs in a Node.js application.
+
+Cron is a [daemon](https://en.wikipedia.org/wiki/Daemon_(computing)), which means that it only needs to be started once, and will 
+lay dormant until it is required. Another example of deamon is the web server, it remains idle until a request for a web page is made. The cron daemon remains idle until the time specified in its configuration files, or crontabs is reached.
+
+
+### Node cron syntax
```

Comment:
This section is very "messy" with a lot of different formats, lists, etc. I marked the different formats etc. with a comment each. Please read through the [preview of the article](https://deploy-preview-489--reflectoring.netlify.app/schedule-cron-job-in-node/) again imagining you are a random reader from the internet and clean it up a bit. Fewer different formats is more!